module/documents/actor.mjs

```diff
@@ -175,6 +175,19 @@ export class PokeroleActor extends Actor {
     return diceCount;
   }
 
+  getDamagePoolForMove(move) {
+    if (move.type !== 'move') {
+      return undefined;
+    }
+
+    let diceCount = 0;
+    if (move.system.category !== 'Support') {
+      diceCount += this.getAnyAttribute(move.system.dmgMod)?.value ?? 0;
+      diceCount += move.system.power;
+    }
+    return diceCount;
```

Comment:
```suggestion
    diceCount += this.getAnyAttribute(move.system.dmgMod)?.value ?? 0;
    diceCount += move.system.power;
    if (move.system.stab) {
      diceCount += 1;
    }
```
The system doesn't usually check for the category to determine if a move is damaging, so this might cause surprises because the "Roll Damage" button could still be visible even though the sheet says that the move doesn't have damage. It checks for `this.system.power || this.system.dmgMod` to determine whether to show the button, so we could get rid of the check entirely and treat the move as non-damaging if `diceCount` stays zero (either show "-" or keep the 0 on the sheet).

I think it'd also be nice to add STAB here. We could get rid of the STAB column then to save some space. As a side note, I should make the STAB damage configurable for modules in case people want to do things like Terastallization bonuses, but it's currently just hardcoded to 1.